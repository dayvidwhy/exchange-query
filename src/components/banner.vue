<style>
</style>

<template>
    <header role="banner" class="row blue-bg">
        <div class="banner-text align-center">
            Made by <a class="banner-link" href="https://davidyoung.tech">https://davidyoung.tech</a>
        </div>
    </header>
</template>

<script>
export default {
    name: "banner"
};
</script>