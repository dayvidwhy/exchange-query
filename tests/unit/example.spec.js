import { shallowMount } from '@vue/test-utils'
import banner from '@/components/banner.vue'

describe('banner.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(banner);
    expect(wrapper).toMatchSnapshot();
  });
});
