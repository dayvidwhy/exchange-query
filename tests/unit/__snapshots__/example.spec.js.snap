// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`banner.vue renders correctly 1`] = `
<header role="banner" class="row blue-bg">
  <div class="banner-text align-center">
    Made by <a href="https://davidyoung.tech" class="banner-link">https://davidyoung.tech</a></div>
</header>
`;
